wagtail/wagtailimages/checks.py

```diff
@@ -0,0 +1,72 @@
+import os
+
+from django.core.checks import register, Warning
+
+from willow.image import Image
+
+
+_has_jpeg_support = None
+_has_png_support = None
+
+
+def has_jpeg_support():
+    global _has_jpeg_support
+
+    if _has_jpeg_support is None:
+        wagtail_jpg = os.path.join(os.path.dirname(__file__), 'check_files', 'wagtail.jpg')
+        succeeded = True
+        f = open(wagtail_jpg, 'rb')
+
+        try:
+            Image.open(f)
```

Comment:
slightly nicer:
```
with open(wagtail_jpg, 'rb') as f:
    Image.open(f)
```

This will close the image file in the event of an exception as well